src/GraphQL.Tests/Types/AutoRegisteringInputObjectGraphTypeTests.cs

```diff
@@ -357,10 +386,10 @@ private class FieldTests
         public IEnumerable<int> NotNullEnumerableNotNullIntField { get; set; } = null!;
         public IEnumerable<int?>? NullableEnumerableNullableIntField { get; set; }
         public IEnumerable<int>? NullableEnumerableNotNullIntField { get; set; }
-        public Tuple<int, string>?[] NotNullArrayNullableTupleField { get; set; } = null!;
-        public Tuple<int, string>[] NotNullArrayNotNullTupleField { get; set; } = null!;
-        public Tuple<int, string>?[]? NullableArrayNullableTupleField { get; set; }
-        public Tuple<int, string>[]? NullableArrayNotNullTupleField { get; set; }
+        public InputTuple<int, string>?[] NotNullArrayNullableTupleField { get; set; } = null!;
+        public InputTuple<int, string>[] NotNullArrayNotNullTupleField { get; set; } = null!;
+        public InputTuple<int, string>?[]? NullableArrayNullableTupleField { get; set; }
+        public InputTuple<int, string>[]? NullableArrayNotNullTupleField { get; set; }
```

Comment:
Note: `Tuple<T1, T2>` is still supported here same as before, but is not supported by the test framework because the auto-registering input type won't generate members for constructor arguments, and there is no input graph type defined for `Tuple<int, string>`.  So to make it easy I just created an `InputTuple<T1, T2>` which has settable properties.  The other option would have been to create an input object graph type definition for `Tuple<T1, T2>`